.github/workflows/ci.yml

```diff
@@ -202,13 +203,14 @@ jobs:
       # would override the config
       - name: Run unit tests with coverage for ${{ matrix.package }}
         if: env.PRIMARY_NODE_VERSION == matrix.node-version && matrix.os == 'ubuntu-latest'
-        run: npx nx test ${{ matrix.package }}
+        run: yarn nx run ${{ matrix.package }}:test
```

Comment:
these changes should happen in #10582 instead (it was only temporarily closed until agreement was met in an issue and needs reopening now)